.github/workflows/release-nightly.yml

```diff
@@ -27,19 +27,27 @@ jobs:
       contents: write
     steps:
       - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: "3.10"
       - uses: actions/download-artifact@v4
         with:
           path: dist
           merge-multiple: true
+      - name: Delete old dev wheels from pypi.anaconda.org
+        run: |
+          python -m pip install anaconda-cli
+          curl --silent "https://api.anaconda.org/package/scipp/scipp" | jq -r '.releases[] | select(.version | contains("dev")) | .full_name' > dev_wheels.txt
```

Comment:
This looks like it lists both conda packages and wheels. Is this correct? Can we go via the PyPI index instead (https://pypi.anaconda.org/scipp/simple/scipp/)?

Reply:
I couldn't find an api end point just for pypi, the `contains("dev")` is to fetch only versions with `dev` in it's name, only the wheels have that in the name.

Reply:
> only the wheels have that in the name.

Sure, currently. It's probably fine. I just want to be extra careful when erasing data so we don't accidentally erase anything that is supposed to be permanent.

Reply:
We can use a new org like https://anaconda.org/scipp-nightly-wheels/ to be safe.

Reply:
I'd be more comfortable with this. But I'd like input from @nvaytet and @SimonHeybrock.

Reply:
:+1: from me for `scipp-nightly-wheels` organization

Reply:
We are just waiting for +1 from @SimonHeybrock right?